app/controllers/api/v1/users_controller.rb

```diff
@@ -102,21 +109,14 @@ def validate_update_params
       end
 
       # If the update is for a user that does not match the current user,
-      # determine if permission should be granted for the update and for
-      # which params.
+      # determine if permission should be granted for the update
       def validate_mismatch
```

Comment:
Should this be called ``validate_id_mismatch``? (The change would apply to the ``validate_match`` function too.)